src/main/java/Quiz/dto/QuestionIdDto.java

```diff
@@ -0,0 +1,45 @@
+package Quiz.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Collection;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class QuestionIdDto {
+    private int id;
+    private String description;
+    private Collection<QuestionOptionIdDto> options;
+
+    public int getId() {
```

Comment:
Please use project lombok so we can be free of basic getters and setters in Moldes and DTOs

        <!-- Object mapper-->
        <dependency>
            <groupId>org.projectlombok</groupId>
            <artifactId>lombok</artifactId>
            <version>1.18.10</version>
        </dependency>